dbt/models/sec10k/core_sec10k__assn_sec10k_filers_and_eia_utilities/schema.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+sources:
+  - name: pudl
+    tables:
+      - name: core_sec10k__assn_sec10k_filers_and_eia_utilities
+        data_tests:
+          - check_row_counts_per_partition:
+              table_name: core_sec10k__assn_sec10k_filers_and_eia_utilities
+              partition_column: null
+        columns:
+          - name: central_index_key
+            tests:
+              - unique
+          - name: utility_id_eia
+            tests:
+              - unique
```

Comment:
I added the per-column unique check here manually. They're already required to be non-null because they're the PK.